images/dotcms/Dockerfile

```diff
@@ -76,6 +76,9 @@ ENV SERVICE_DELAY_DEFAULT_MAX 30
 
 #VOLUME /data/local
 
+## JDK upgrade does not take first run, needs to runs
+RUN  apk upgrade
```

Comment:
Isn't this already running at line 36?

Reply:
comment says it needs two runs

Reply:
that is right - for some reason, it does not update the java version the first time.  It says it does, but when you run `java -version` it is still the old `1.8.212`